### DISCUSSION TITLE
Wet bulb globe temperature?

### DISCUSSION THREAD

**Discussion by chillgamesh** [breezy-weather/breezy-weather#1254] [Ideas]
### Describe your idea

I'd like to see WBGT added to the forecast/cards. It's a relatively simple calculation that takes into account sun exposure and wind to give a more accurate safety indicator for outdoor exercise and heavy activity, but it's not often directly reported on most weather sites. 

### Other details

https://www.weather.gov/tsa/wbgt

### Acknowledgements

- [X] I have searched the [existing issues](https://github.com/breezy-weather/breezy-weather/issues) and [existing discussions](https://github.com/breezy-weather/breezy-weather/discussions/categories/ideas) and this is a new ticket, **NOT** a duplicate or related to another open OR closed issue or discussion.
- [X] The idea I’m suggesting is not in the [list of features that will not be implemented](https://github.com/breezy-weather/breezy-weather/blob/main/README.md#features-that-will-not-be-implemented).
- [X] I have updated the app to **[version 5.2.7 or later](https://github.com/breezy-weather/breezy-weather/releases/latest)**. Even if I’m getting updates from F-Droid default repo, I don’t need to uninstall/reinstall to update.
- [X] I wrote my request in English, and any screenshot I attached are also in English (language can be changed from Settings > Appearance). I can write in my native language under the English text, if necessary.
- [X] I understand that this app is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY, so I may not get a positive reply to my request, if any at all.

**Comment by papjul** [comment]
I'd not say it's a simple formula. We don't directly have the sun angle, and it depends on many parameters, some which might be missing from the source. It's also missing air quality if you are looking for a safety indicator for outdoor practice. I will investigate the possibility further later

**Comment by papjul** [comment]
This will not be worked on, but feel free to get the necessary data from a `nodomain.freeyourgadget.gadgetbridge.ACTION_GENERIC_WEATHER` receiver and do your own computing on your app
